Add Orus Group DD-IMPORTER integration (#2611)
DataDog/integrations-extras: orus_group_dd_importer/README.md

# Orus Group DD-IMPORTER

## Overview

> As enterprises migrate to cloud platforms like AWS, monitoring dynamic and distributed resources becomes increasingly complex. Traditional monitoring tools, designed for static environments, struggle to handle the rapid creation, modification, and termination of cloud resources, leading to issues like inconsistent visibility, operational inefficiency, and alert fatigue. AWS Cloudwatch, while offering basic monitoring, has limitations such as single-threshold alarms, no support for multi-cloud environments, a cumbersome interface, and required AWS access, making it inadequate for enterprise cloud operations. The Orus Group's Cloudwatch to Datadog Alarm Importer (DD-Importer) addresses these challenges by deploying monitors into Datadog using structured tagging and automatic deployment to produce optimized monitoring efficiency. This innovative product automatically deploys monitors that match your AWS resources with no coding and no API keys required.  Using our app allows you to setup your monitors in minutes.

Reviewer: ```suggestion
As enterprises migrate to cloud platforms like AWS, monitoring dynamic and distributed resources becomes increasingly complex. Traditional monitoring tools designed for static environments struggle to manage the rapid creation, modification, and termination of cloud resources, resulting in inconsistent visibility, operational inefficiency, and alert fatigue. 

While AWS CloudWatch provides basic monitoring, it has limitations such as single-threshold alarms, no support for multi-cloud environments, a cumbersome interface, and required AWS access, making it inadequate for enterprise cloud operations. 

The Orus Group's CloudWatch to Datadog Alarm Importer (DD-Importer) addresses these challenges by deploying monitors into Datadog using structured tagging and automated deployment for optimized monitoring efficiency. This innovative product automatically deploys monitors that match your AWS resources with no coding or API keys required.  With our app, you can set up your monitors in minutes.
```

## Setup

> DD-IMPORTER is designed for use with no coding and no API keys required.  Our app provides uses with pre-designed monitors for many AWS services.  You can edit and tailor the recommended monitors to your desired settings using Microsoft Excel.  When you're ready, just click the "Generate Monitors" button and upload your monitors directory into Datadog through the DD-IMPORTER app.  

Reviewer: ```suggestion
DD-IMPORTER is designed for use without coding or API keys.  Our app provides users with pre-designed monitors for various AWS services.  You can customize the recommended monitors to your desired settings using Microsoft Excel.  When you're ready, click the "Generate Monitors" button and upload your monitors directly into Datadog through the DD-IMPORTER app.  
```

>  NOTE: we do not save your monitors or ever know your login credentials. 

Reviewer: ```suggestion
**Note**: DD-IMPORTER does not save your monitors or have access to your login credentials. 
```

## Uninstallation

> There is nothing to install or un-install.  We do not save your monitors or ever know your login credentials. 

Reviewer: ```suggestion
No installation or uninstallation is required.
```
